stdlib/Artifacts/src/Artifacts.jl

```diff
@@ -541,7 +541,9 @@ function _artifact_str(__module__, artifacts_toml, name, path_tail, artifact_dic
     meta = artifact_meta(name, artifact_dict, artifacts_toml; platform)
     if meta !== nothing && get(meta, "lazy", false)
         if lazyartifacts isa Module && isdefined(lazyartifacts, :ensure_artifact_installed)
-            nameof(lazyartifacts) === :Pkg && Base.depwarn("using Pkg instead of using LazyArtifacts is deprecated", :var"@artifact_str", force=true)
+            if nameof(lazyartifacts) in (:Pkg, :Artifacts)
+                Base.depwarn("using Pkg instead of using LazyArtifacts is deprecated", :var"@artifact_str", force=true)
```

Comment:
Why `force=true`?

Reply:
I believe Jameson put that there so that even if users have `depwarn`s normally silenced, it will show up for them.

I don't think we can remove it, but I think usage of this will hamper future compiler work, so we want to try and eliminate as much of it from the ecosystem as possible.

Reply:
Fair enough

Reply:
Kind of defeats the purpose with deprecation warnings off by default though.